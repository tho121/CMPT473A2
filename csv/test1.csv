
hello,32,my,15,name,13
is,49817,albert,12308,21,heink