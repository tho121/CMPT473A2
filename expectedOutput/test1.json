[
{"0":"hello","1":"32","2":"my","3":"15","4":"name","5":"13"},
{"0":"is","1":"49817","2":"albert","3":"12308","4":"21","5":"heink"}
]
